pyproject.toml

```
@@ -59,9 +59,11 @@ python-crontab = "3.2.0"
 wallycore = {version = "1.3.0", optional = true}
 # needed for breez funding source
 breez-sdk = {version = "0.5.2", optional = true}
+breez-sdk-liquid = {version = "0.3.0", optional = true}
 
 [tool.poetry.extras]
 breez = ["breez-sdk"]
+breez_liquid = ["breez-sdk-liquid"]
```

let's use dash instead of underscore
```suggestion
breez-liquid = ["breez-sdk-liquid"]
```